### DISCUSSION TITLE
Welcome to microfeed Discussions!

### DISCUSSION THREAD

**Discussion by wenbinf** [microfeed/microfeed#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hailspuds** [comment]
Fantastic project. Really interesting and great work. Thank you!!

**Comment by jrcreative** [comment]
Great work! I'm currently fascinated by all the services that Cloudflare is offering at the edge. I'm using the microfeed JSON feed as a data source for my Hugo site. Basically a mini headless CMS. It's very cool!

**Reply by hailspuds** [reply]
Oh that's a great idea! Anywhere I can have a look/read about how you've hooked it up?

**Reply by wenbinf** [reply]
One of intended usecases of microfeed is headless CMS :)

You can fetch either json or rss for feed or item - just add /rss or /json to the end of an url:
* https://www.microfeed.org/json/
* https://www.microfeed.org/rss/
* https://www.microfeed.org/i/introducing-microfeed-a-self-hosted-open-source-cms-on-cloudflare-open-alpha-uhbQEmArlC2/json/
* https://www.microfeed.org/i/introducing-microfeed-a-self-hosted-open-source-cms-on-cloudflare-open-alpha-uhbQEmArlC2/rss/

**Reply by jrcreative** [reply]
> Oh that's a great idea! Anywhere I can have a look/read about how you've hooked it up? 

It's really simple with Hugo. Just feed it a url and you can pull it into your template. https://gohugo.io/templates/data-templates/#get-remote-data
